source/plugin-settings/obox-storage-plugin.rst

```diff
@@ -137,3 +137,21 @@ There are no migrated POP3 UIDLs. Don't try to look them up in any situation. No
 
 By default doing changes to folders (e.g. creating or renaming) uploads changes immediately to object storage. If this setting is enabled, the upload happens sometimes later (within metacache_upload_interval).
 
+.. _plugin-obox-storage-settting_obox_allow_copy_between_users:
+
+``obox_allow_copy_between_users``
+---------------------------------
+
+.. versionadded:: v2.3.11
+
+If set to yes this setting enables sharing mail-objects in storage between
+users. If enabled it can reduce the storage usage, for example mails
+with multiple recipients, will be only stored once, this also affects expunged
+mails. The deduplication is currently only supported with ``obox_fs`` using
+fs-dictmap. Enabling this setting requires a change to the map configuration see
+:ref:`reference_counted_mapping_allow_copy_between_users`.
+
+
+.. Warning:: When using this setting together with per user fs-crypt mind that
+             copying between users with different encryption keypairs cannot
+             use deduplication.
```

Comment:
I think could just remove "per user".

Also maybe a comment that with mail_crypt it can never deduplicate? I think? Would be nice to verify also :)

Reply:
Yes, i tried to make it clear that as soon as keys differ (folder, user or global) it cannot deduplicate.